fedmsg_meta_fedora_infrastructure/tests/pkgdb.py

```diff
@@ -366,7 +366,7 @@ class LegacyTestPkgdbBranchClone(Base):
     }
 
 
-class TestPkgdbCritpathUpdate(Base):
+class TestLegacyPkgdbCritpathUpdate(Base):
```

Comment:
Yes.  That one is tested here:  https://github.com/fedora-infra/fedmsg_meta_fedora_infrastructure/blob/feature/pkgdb-usernames-kludge-removal/fedmsg_meta_fedora_infrastructure/tests/pkgdb.py#L1483-L1499